Source/WebKit/WebProcess/WebPage/Cocoa/TextAnimationController.mm

```diff
@@ -93,17 +93,23 @@
     return corePage->contextRangeForSessionWithID(sessionID);
 }
 
-std::optional<WebCore::SimpleRange> TextAnimationController::contextRangeForTextAnimationType(const WTF::UUID& uuid) const
+std::optional<WebCore::SimpleRange> TextAnimationController::contextRangeForTextAnimationID(const WTF::UUID& animationUUID) const
 {
-    if (auto sessionRange = contextRangeForSessionWithID(uuid))
+    // FIXME: This shouldn't be needed any longer, but still is.
```

Comment:
>but still is
why? this seems like it can't ever return a non-nil value if you're using an animation id for a session id?

Reply:
Oh this is leftover from when I was trying to figure out the crashes, this should be removed,  I think.